crt_portal/cts_forms/templates/forms/snippets/pagination.html

```diff
@@ -0,0 +1,38 @@
+<nav class="usa-pagination" role="navigation" aria-label="Pagination">
+  <ol class="pagination">
+  <!-- first arrow button -->
+  {% if page_format.has_previous %}
+    <li><a href="?page={{ page_format.previous_page_number }}&per_page={{ page_format.per_page }}"aria-hidden="true">&laquo;</a></li>
+  {% else %}
+    <li class="disabled" aria-hidden="true">&laquo;</li>
+  {% endif %}
+  <!-- pagination buttons-->
+  {% if page_format.show_first %}
+    <li><a href="?page=1&per_page={{ page_format.per_page }}" role="button" aria-label="Go to page 1">1</a></li>
+  {% endif %}
+  {% if page_format.first_ellipsis %}
+    <li><span class="ellipsis" aria-hidden='true'>...</span></li>
+  {% endif %}
+
+  {% for linkpage in page_format.page_numbers %}
+    {% ifequal linkpage page_format.page %}
+       <li class="is-active" aria-label="Current page, page {{ page_format.page }}"><b>{{ page_format.page }}</b></li>
```

Comment:
I think this should have `aria-current="true"` set: 

https://www.a11ymatters.com/pattern/pagination/#3--indicate-which-element-is-currently-active

If you want to update this now, excellent, or feel free to leave it to me when I pick up styling if that's easier. 🚢